### DISCUSSION TITLE
Indentation lost on accepting inline suggestions in VS 2022

### DISCUSSION THREAD

**Discussion by hungFl** [community/community#90080] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi,

After a recent update to either Visual Studio 2022(17.8.5) or Copilot extension(1.149), I've encountered an issue: accepting an inline suggestion results in the loss of indentation, even though the suggestions themselves display with correct indentation.

https://github.com/community/community/assets/44134554/1b108a48-dd85-4dfa-9949-36f9cb64e7a9

Any advice on fixing this? Thanks in advance!

**Comment by ayim** [comment, marked as answer]
Hey! First time I'm seeing this specific issue.
Please file a bug using this mechanism within Visual Studio so we can get started at diagnosing.
![image](https://github.com/community/community/assets/10951869/428b9d88-9484-44f4-a6e4-1a8b924f6f99)


--
Aaron Yim
Product Manager for Copilot in vS
